src/state/actions/notes/index.js

```diff
@@ -0,0 +1,20 @@
+import axiosWithAuth from '../../../utils/axiosWithAuth';
```

Comment:
These look like changes unrelated to the PR (readme file needs changes). Was this part of another branch being worked on?

Reply:
Yeah my mistake was adding it to this pr. I Didn't realize that it would be attached to the push

Reply:
I think I fixed it. This stackoverflow article was helpful:
https://stackoverflow.com/questions/36168839/how-to-remove-commits-from-a-pull-request
I had to remove a commit that seemed out of place